scripts/lint-lists.py

```diff
@@ -21,6 +21,20 @@
 LEGACY_CATEGORY_CODES = "00-LEGEND-category_codes.csv"
 COUNTRY_CODES = "00-LEGEND-country_codes.csv"
 
+def is_valid_date(date):
+    try:
+        y, m, d = date.split('-')
```

Comment:
Why not just use: `datetime.datetime.strptime(date, "%Y-%M-%d")`?

Reply:
Mainly because it returns a datetime.date rather than a datetime.datetime.  There's no datetime.date.strptime method.  Also, I wasn't sure whether you needed this code to work with really ridiculously old Python (datetime.datetime.strptime was added in 2.5).

I'm happy to change it to use strptime if you would prefer.  Please let me know, then I'll rebase it.

Reply:
I see. Yeah legacy python support is not a concern as this is just meant to run on the CI. I would change it to using standard python `datetime` parsing functions to make it more robust to other forms of invalid dates (ex. `2018-02-31`).